src/vs/workbench/contrib/chat/browser/chatEditing/notebook/chatEditingNotebookCellEntry.ts

```diff
@@ -241,7 +241,7 @@ export class ChatEditingNotebookCellEntry extends ObservableDisposable {
 	private async _acceptHunk(change: DetailedLineRangeMapping): Promise<boolean> {
 		this._isEditFromUs = true;
 		try {
-			if (!this._diffInfo.get().changes.includes(change)) {
+			if (!this._diffInfo.get().changes.filter(c => c.modified.equals(change.modified) && c.original.equals(change.original)).length) {
```

Comment:
@DonJayamanne - This change is for the case that we are accepting a collection of changes - each one will cause a diff recompute and the changes will no longer be the same object.
Maybe better to pass in an array of changes, though I think that would require modifying the IDocumentDiff2 interface.

Reply:
@amunger Would be good to add this comment into source